src/TaskList.test.js

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import * as requests from "./utils/utils";
+import { render, screen } from "@testing-library/react";
+import TaskList from "./components/TaskList";
+import { BrowserRouter } from "react-router-dom";
+import { act } from "react-dom/test-utils";
+
+describe("TaskList", () => {
+	const fakeTasks = [
+		{
+			id: 9,
+			task_name: "Book tickets to see Spiderman",
+			description: "at the Odeon",
+			completed: true,
+			created_at: "2021-11-24T10:07:04.696Z",
+		},
+	];
+	it("a request is made with the right parameter", async () => {
+		jest.spyOn(requests, "fetchData").mockResolvedValue(fakeTasks);
```

Comment:
this line is repeated across all tests, could be extracted to a `beforeEach` block.

Reply:
Done 👍